stockflux-securities-master/src/components/InputForm.js

```diff
@@ -133,6 +156,11 @@ const InputForm = ({ match }) => {
           </div>
         </form>
       )}
+      <ul>
```

Comment:
The `Alert` component is now a list, but you're using it here as if it still represents a single item. You'll end up rendering a list, within a list.

Reply:
Yep, forgot to remove that. Gone now.